packages/aws-cdk/lib/api/cxapp/exec.ts

```diff
@@ -75,14 +77,40 @@ export async function execProgram(aws: SdkProvider, config: Configuration): Prom
   env[cxapi.OUTDIR_ENV] = outdir;
 
   // Send version information
-  env[cxapi.CLI_ASM_VERSION_ENV] = cxapi.CLOUD_ASSEMBLY_VERSION;
```

Comment:
Could this break anyone relying on this environment variable?

Reply:
> The "error:" prefix is redundant. Exceptions are printed with "Error: ".

What error prefix? 

> Could this break anyone relying on this environment variable?

God i hope not. But yeah you're right, i'll address it.

Reply:
resurrected